File: Problem 327P1/0327_Problem_1.go
// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 15 ms

func maximumCount(nums []int) int {
    var pos, neg int = 0, 0
    for _, e := range nums {
	    if e > 0 {
            pos++
        } else if e < 0 {
            neg++
        }
    }
    if pos > neg {
        return pos
    } else {
        return neg
    }
}

File: Problem 327P1/0327_Problem_1.php
<?php

declare(strict_types=1);

// Problem Link: https://leetcode.com/problems/maximum-count-of-positive-integer-and-negative-integer/
// LeetCode Benchmark: 39 ms

function maximumCount(array $nums): int {
    $pos = 0;
    $neg = 0;
    foreach ($nums as $e) {
        if ($e > 0) {
            $pos++;
        } elseif ($e < 0) {
            $neg++;
        }
    }
    return max($pos, $neg);
}
